data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/reconciler/impl/ResourcesReconcilerImpl.java

```diff
@@ -54,13 +55,12 @@ private ResourcesReconcilerImpl(
   }
 
   @Override
-  public Future<Void> reconcile(
-    Collection<DataPlaneContract.Resource> newResources) {
-    final Map<String, DataPlaneContract.Resource> newResourcesMap = new HashMap<>(
-      newResources
-        .stream()
-        .collect(Collectors.toMap(DataPlaneContract.Resource::getUid, Function.identity()))
-    );
+  public Future<Void> reconcile(final Collection<DataPlaneContract.Resource> newResources) {
+    final Map<String, DataPlaneContract.Resource> newResourcesMap =
+        new HashMap<>(
+            newResources.stream()
+                .collect(
+                    Collectors.toMap(DataPlaneContract.Resource::getUid, Function.identity())));
```

Comment:
That style is so bad, with all the parenthesis at the end. I want lines with one close parens indented.

Reply:
google-java-style format code in this way, no way to change it by design.

Reply:
There are no other maintained tools like that out there afaik.

Reply:
Oh really? This is really a no go for me.... Can't you just disable the parenthesis formatting?

Reply:
Nope, we can't.

Reply:
```
                    Collectors.toMap(DataPlaneContract.Resource::getUid, Function.identity())));
```
is ugly to read - no?